codespell_lib/data/dictionary.txt

```diff
@@ -10575,7 +10656,10 @@ fonction->function
 fonctional->functional
 fonctionalities->functionalities
 fonctioning->functioning
+fonctionnalies->functionalities
 fonctionnalities->functionalities
+fonctionnality->functionality
+fonctionnaly->functionality, functionally,
```

Comment:
Can you swap the suggestions, I think the latter is the more likely intention from the typo.